tools/diagnostics/base_tech_bundle/cassandra_bundle.py

```diff
@@ -0,0 +1,49 @@
+import logging
+import retrying
+
+import sdk_cmd
+
+from diagnostics.base_tech_bundle import BaseTechBundle
+
+logger = logging.getLogger(__name__)
+
+DEFAULT_RETRY_WAIT = 1000
+DEFAULT_RETRY_MAX_ATTEMPTS = 5
+
+
+class CassandraBundle(BaseTechBundle):
+    @retrying.retry(
+        wait_fixed=DEFAULT_RETRY_WAIT, stop_max_attempt_number=DEFAULT_RETRY_MAX_ATTEMPTS
+    )
+    def task_exec(self, task_id, cmd):
+        full_cmd = " ".join(
+            [
+                "export JAVA_HOME=$(ls -d ${MESOS_SANDBOX}/jdk*/jre/) &&",
+                "export TASK_IP=$(${MESOS_SANDBOX}/bootstrap --get-task-ip) &&",
+                "CASSANDRA_DIRECTORY=$(ls -d ${MESOS_SANDBOX}/apache-cassandra-*/) &&",
+                cmd,
+            ]
+        )
+
+        return sdk_cmd.marathon_task_exec(task_id, "bash -c '{}'".format(full_cmd))
+
+    def create_nodetool_status_file(self, task_id):
```

Comment:
I had pointed this out already in https://github.com/mesosphere/dcos-commons/pull/2511#discussion_r206405469 - can you name these `download_` or `fetch_` or `retrieve_` rather than `create_`?
Creating a bundle sounds OK, but creating a status file sounds like it's our status we write, while in fact we just retrieve data from elsewhere.

Reply:
For `create_log_files` I agree that `download_` or `fetch_` makes more sense, but here the code isn't downloading/fetching a nodetool status file, it's creating a file based on a command output, `nodetool status`.

ffb0d9dca1c4492fa9ff4f1f8f31ff66403bfbd5